docs/template_plugin/tests/functional/op_reference/scatter_update.cpp

```diff
@@ -0,0 +1,914 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph/ngraph.hpp>
+#include <shared_test_classes/base/layer_test_utils.hpp>
+#include <vector>
+
+#include "base_reference_test.hpp"
+#include "ngraph_functions/builders.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
+using namespace reference_tests;
+
+namespace reference_tests {
+
+namespace {
+
+// ---------------------- V3 ------------------------------
+
+struct ScatterUpdate3Params {
+    Tensor data;
+    Tensor indices;
+    Tensor updates;
+    Tensor axis;
+    Tensor expected;
+};
+
+struct Builder : ParamsBuilder<ScatterUpdate3Params> {
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, data);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, indices);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, updates);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, axis);
+    REFERENCE_TESTS_ADD_SET_PARAM(Builder, expected);
+};
+
+class ReferenceScatterUpdate6LayerTest : public testing::TestWithParam<ScatterUpdate3Params>, public CommonReferenceTest {
+public:
+    void SetUp() override {
+        auto params = GetParam();
+        function = CreateFunction(params.data.shape, params.indices.shape, params.updates.shape, params.axis.shape,
+                                  params.data.type, params.indices.type, params.axis.type);
```

Comment:
nitpick: Maybe we can reduce the number of parameters if just use `params` as an argument.